msi/msi_test.go

```diff
@@ -161,3 +162,21 @@ func TestCanGetMsiForStoragetWithObjectId(t *testing.T) {
 		t.Fatal(err.Error())
 	}
 }
+
+func TestCanGetImdsVariable(t *testing.T) {
+
+	//without setting env var - should return metadataURL
+	url := GetMetadataIdentityURL()
+	if url != metadataIdentityURL {
+		t.Fatal("mismatch URLs")
+	}
+
+	//setting env var - should return test (or whatever var is set to)
+	os.Setenv(identityEnvVar, "test")
```

Comment:
Move it into a variable within the test, also use a constant that looks like a URL